developer-docs-site/docs/guides/getting-started.md

```diff
@@ -61,9 +61,10 @@ Start by cloning the `aptos-core` GitHub repo from [[GitHub](https://github.com/
 
 Now your basic Aptos development environment ready.
 
-## Plugins
+## Move plugins
 
 - A [Move language syntax plugin](https://marketplace.visualstudio.com/items?itemName=damirka.move-syntax) for Visual Studio Code.
+- A [Visual Studio Code plugin for Move, called move-analyzer](https://marketplace.visualstudio.com/items?itemName=move.move-analyzer).
```

Comment:
```suggestion
- [Move-Analyzer for VS Code](https://marketplace.visualstudio.com/items?itemName=move.move-analyzer) - supports advanced code navigation and syntax highlighting
```

Reply:
The term `move-analyzer` is the name of the plugin, and we should use the term as it is. Changing the above suggested `Move-Analyzer` to `Move analyzer` so now this is just a label.